test_runner/scenario_test_runner/test/scenario/autoware-simple-test.yaml

```diff
@@ -0,0 +1,167 @@
+ScenarioModifiers:
+  ScenarioModifier:
+    - { name: HOGE, start: 0, step: 2, stop: 5 }
+    # - { name: FUGA, start: 1, step: 0.5, stop: 2 }
+OpenSCENARIO:
+  FileHeader:
+    revMajor: 1
+    revMinor: 0
+    date: '2021-02-01T08:31:04.600Z'
+    description: ''
+    author: ''
```

Comment:
What is the purpose of this newly added scenario?

Reply:
It was added for testing purpose, to validate yaml format file with reach position different from final destination. Now removed